src/openMVG/cameras/Camera_Intrinsics.hpp

```diff
@@ -128,6 +133,7 @@ struct IntrinsicBase
   {
     ar(cereal::make_nvp("width", _w));
     ar(cereal::make_nvp("height", _h));
+    ar(cereal::make_nvp("serialNumber", _serialNumber));
```

Comment:
@eprana  @fabiencastan In order to avoid to break backcompatibility with the reconstructions we have already, can we surround it with a try catch and in case initialize it to a default value / empty string?
Something like this: 
https://github.com/poparteu/openMVG/blob/popart_develop/src/openMVG/sfm/sfm_landmark.hpp#L95